crates/pixi_utils/src/executable_utils.rs

```diff
@@ -2,46 +2,52 @@ use std::path::Path;
 
 /// Strips known Windows executable extensions from a file name.
 pub(crate) fn strip_windows_executable_extension(file_name: String) -> String {
-    let file_name = file_name.to_lowercase();
-    // Attempt to retrieve the PATHEXT environment variable
+    // Create lowercase version for comparison
+    let lowercase_name = file_name.to_lowercase();
+
+    // Get extensions list
     let extensions_list: Vec<String> = if let Ok(pathext) = std::env::var("PATHEXT") {
         pathext.split(';').map(|s| s.to_lowercase()).collect()
     } else {
-        // Fallback to a default list if PATHEXT is not set
         tracing::debug!("Could not find 'PATHEXT' variable, using a default list");
         [
-            ".COM", ".EXE", ".BAT", ".CMD", ".VBS", ".VBE", ".JS", ".JSE", ".WSF", ".WSH", ".MSC",
-            ".CPL",
+            ".com", ".exe", ".bat", ".cmd", ".vbs", ".vbe", ".js", ".jse", ".wsf", ".wsh", ".msc",
```

Comment:
The PATHEXT variable also uses uppercase so I would keep it that way.

Reply:
I removed it because the strings were followed by a `to_lowercase`, but sure, I can keep it.

Reply:
Since this keeps current behavior, let's leave it as is